web/src/click_handlers.js

```diff
@@ -894,6 +893,35 @@ export function initialize() {
         }
     });
 
+    // COMPOSE BUTTON TOOLTIPS
+
+    const compose_buttons = tippy(
+        ".compose_control_button_container, .compose_control_button[data-tooltip-template-id], .compose_control_button[data-tippy-content]",
+        {
+            trigger: "mouseenter",
+            placement: "top",
+        },
+    );
+
+    createSingleton(compose_buttons, {
+        onTrigger(instance, event) {
+            const templateId = event.currentTarget.dataset.tooltipTemplateId;
+            if (templateId) {
+                const templateElement = document.querySelector(`#${templateId}`);
+                if (templateElement) {
+                    const parsedHTML = ui_util.parse_html(templateElement.innerHTML);
+                    instance.setContent(parsedHTML);
+                }
+            }
+
+            if (event.currentTarget.classList.contains("disabled-on-hover")) {
+                instance.setProps({delay: 0});
```

Comment:
Is there a reason to not use `INSTANT_HOVER_DELAY` from `tippyjs.ts` here? Or `INTERACTIVE_HOVER_DELAY` inside the else block? We should avoid these kinds of one-off scalar values, whenever possible. Over time, they can make the UI feel inconsistent.

Reply:
Yes there are various dividers between the compose control buttons which re-triggers the `INSTANT_HOVER_DELAY` whenever we try to smoothly transition from one button to another. 
Due to which the `createSingleton()` function is re-triggered everytime a divider is encountered.